package.json

```diff
@@ -78,7 +79,7 @@
     "lodash": "^4.17.10",
     "multiselect": "~0.9.12",
     "patternfly": "^3.42.0",
-    "patternfly-react": "2.5.1",
+    "patternfly-react": "^2.11.3",
```

Comment:
@ekohl can you confirm these changes? thanks

Reply:
any decision ? :)